attributes/default.rb

```diff
@@ -1,7 +1,7 @@
-default['pgd']['pgd_path'] = "/home/vagrant/pgd"
-default['pgd']['virtualenv_path'] = "/home/vagrant/vrt"
-default['pgd']['user'] = "vagrant"
-default['pgd']['group'] = "vagrant"
+default['pgd']['pgd_path'] = "/home/centos/pgd"
```

Comment:
@jordane Does this go in it's own code block under the main driver_config or under platforms driver config or under provisioner's attributes? (I'll look around for some examples)